### PR TITLE
Homepage: Why SimplQ Section

### PR DESCRIPTION
**Pull request by daltonfury42**
image

### CODE REVIEW

File: simplq/src/styles/homePage.module.scss [hunk 1]
```diff
@@ -28,41 +43,65 @@
...
   }
+  // Position needs to be set for parent so that corona particles can be
+  // positioned absolute to this div
+  position: relative;
+  .corona-1 {
+    height: 7rem;
+    position: absolute;
+    top: 0rem;
+    right: 15vw;
+  }
```
Comment (maaverik) [inline simplq/src/styles/homePage.module.scss:94]: Did you mean rem here?

**Comment by maaverik** [comment]
Screenshot from 2020-09-20 01-12-25

Add a little padding here too.

**Comment by maaverik** [comment]
Screenshot from 2020-09-20 01-15-46

It looks like one of the css changes for the height have messed up the sizes of the admin and join pages.

**Comment by daltonfury42** [comment]
> Screenshot from 2020-09-20 01-12-25
> 
> Add a little padding here too.

It will anyway be completely redesigned in https://github.com/SimplQ/simplQ-frontend/issues/211 



> Screenshot from 2020-09-20 01-15-46
> 
> It looks like one of the css changes for the height have messed up the sizes of the admin and join pages.

I'm not worried, I'm like, we can fix all these when we take up that particular section

File: simplq/src/styles/homePage.module.scss (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/styles/homePage.module.scss:94]: vw, because I wanted it to be relative to the viewport width.

Comment (maaverik) [reply simplq/src/styles/homePage.module.scss:94]: Okay

**Review by maaverik — APPROVED** [review]

**Pull request merged**